### PR TITLE
VORTEX-6071: Adding all coordinate formats to the dot popup

### PR DESCRIPTION
**Pull request by rkausch**
Fixes #6071

## Proposed Changes
  - Adds DD, DMS, DDM and MGRS, plus altitude to the dot popup.

### CODE REVIEW

File: open-sphere-base/overlay/src/main/java/io/opensphere/overlay/CursorPositionPopupManager.java [hunk 1]
```diff
@@ -11,28 +11,49 @@
...
+    /**
+     * Supplier for the dialog parent. Package visibility to prevent synthetic
+     * accessors.
+     */
+    final Supplier<? extends JFrame> myDialogParentSupplier;
 
-    /** The MGRS text. */
-    private String myMGRSText;
+    /**
+     * a flag to inform the popup that an elevation provider is present. Package
```
Comment (Xenotide) [inline open-sphere-base/overlay/src/main/java/io/opensphere/overlay/CursorPositionPopupManager.java]: Capitalization.

File: open-sphere-base/overlay/src/main/java/io/opensphere/overlay/CursorPositionTransformer.java [hunk 2]
```diff
@@ -200,8 +196,9 @@ public void setLabels(String text1, LatLonAlt latLonAlt, boolean hasElevationPro
         if (isOpen())
         {
             myCursorPositionPanel.setLabels(text1, latLonAlt, hasElevationProvider);
-            myCursorPositionPopupManager.setLabels(myCursorPositionPanel.getLatText(), myCursorPositionPanel.getLonText(),
-                    myCursorPositionPanel.getAltText(), myCursorPositionPanel.getMGRSText());
+            myCursorPositionPopupManager.setLocation(latLonAlt, hasElevationProvider);
+//            myCursorPositionPopupManager.setLabels(myCursorPositionPanel.getLatText(), myCursorPositionPanel.getLonText(),
```
Comment (Xenotide) [inline open-sphere-base/overlay/src/main/java/io/opensphere/overlay/CursorPositionTransformer.java]: Commented code.

File: open-sphere-base/overlay/src/main/java/io/opensphere/overlay/CursorPositionPopupManager.java (hunk 1 above)
Comment (rkausch) [reply open-sphere-base/overlay/src/main/java/io/opensphere/overlay/CursorPositionPopupManager.java]: fixed

File: open-sphere-base/overlay/src/main/java/io/opensphere/overlay/CursorPositionTransformer.java (hunk 2 above)
Comment (rkausch) [reply open-sphere-base/overlay/src/main/java/io/opensphere/overlay/CursorPositionTransformer.java]: fixed

**Pull request merged**